Python-packages/covidcast-py/covidcast/covidcast.py

```diff
@@ -415,6 +425,56 @@ def _fetch_epidata(data_source: str,
     return dfs
 
 
+def _async_fetch_epidata(data_source: str,
+                         signal: str,  # pylint: disable=W0621
+                         start_day: date,
+                         end_day: date,
+                         geo_type: str,
+                         geo_value: Union[str, Iterable[str]],
+                         as_of: date,
+                         issues: Union[date, tuple, list],
+                         lag: int) -> Union[pd.DataFrame, None]:
+    """Fetch data for a single geo asynchronously.
+
+    signal() wraps this to support fetching data over an iterable of
```

Comment:
```suggestion
    signal() wraps this to support fetching data over an iterable set of
```
?

Reply:
This reminded me the docstring is a bit out of date, updated it.